x/auth/vesting/cmd/vestcalc/README.md

```diff
@@ -32,20 +32,36 @@ tool for other options.
 
 ## Writing a schedule
 
-When the `--write` flag is set, the tool will write a schedule in JSON to
-stdout. The following flags control the output:
+When the `--write` flag is set, the tool will write a schedule to stdout.
+The format is the the format expected by the `create-periodic-vesting-account`
+or `create-clawback-vesting-account` cli commands, namely a JSON
```

Comment:
JSON implies text. However, from all the format-related comments, it could be confusing to have it spread out. I've added a single format section with an example that I hope makes it more clear.